blocks/init/components/google-rich-snippets/google-rich-snippets.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Google Rich Snippets
+ *
+ * @package Eightshift_Boilerplate\Views\Parts
+ *
+ * @since 1.0.0
+ *
+ * TODO: Create this as a class with attributes.
+ */
+
+use Eightshift_Boilerplate\Core\Config;
+use Eightshift_Libs\Manifest\Manifest;
+
+$logo_img = apply_filters( Config::get_config( Manifest::MANIFEST_ITEM_FILTER_NAME ), 'logo.svg' );
+?>
+
+<!-- Google Rich Snippets -->
+<script type="application/ld+json">
+  {
```

Comment:
Spacing / code alignment seems weird in the rich snippets section, could you fix it up a bit?

Reply:
yea this :D this was done 3 years ago in 2 minutes. I will rewrite it in a more sensible way after this PR. So just leave it like this for now